src/test/java/org/folio/rest/impl/AbstractInstanceRecordsAPIImpl.java

```diff
@@ -0,0 +1,26 @@
+package org.folio.rest.impl;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.sqlclient.Tuple;
+import java.util.Collections;
+import java.util.Map;
+import java.util.function.Supplier;
+import javax.ws.rs.core.Response;
+import org.folio.rest.RestVerticle;
+import org.folio.rest.api.StorageTestSuite;
+
+public class AbstractInstanceRecordsAPIImpl extends AbstractInstanceRecordsAPI {
+  private static Map<String,String> okapiHeaders = Collections.singletonMap(
+      RestVerticle.OKAPI_HEADER_TENANT, StorageTestSuite.TENANT_ID);
+
+  public static void fetchRecordsByQuery(String sql, RoutingContext routingContext,
```

Comment:
As the `fetchRecordsByQuery` method is only used in one test class, it can be moved there (if the underlying method is made public) and this class removed.